src/test/java/com/upserve/uppend/lookup/LongLookupTest.java

```diff
@@ -26,7 +82,7 @@ public void testKeysDepth1() throws Exception {
     public void testKeysDepth2() throws Exception {
         Path path = Paths.get("build/test/lookup-metadata-test/LongLookupTest");
         SafeDeleting.removeDirectory(path);
-        LongLookup longLookup = new LongLookup(path, 257, LongLookup.DEFAULT_WRITE_CACHE_SIZE);
```

Comment:
These numbers were chosen to cause specific directory nesting depths - this change breaks these tests, please do not merge.